src/test/unit/specific_feature_tests/skins.js

```diff
@@ -0,0 +1,36 @@
+/* jshint maxlen: 100 */
+/* global -$,
+    QUnit,
+    equal,
+    module,
+    notEqual,
+    test,
+    prepareUnitTestModule
+*/
+"use strict";
+
+module(
+    "skin-ui-overrides",
+    {
+        setup: function () {
+            prepareUnitTestModule({
+                options: {skin: 'basic'}
```

Comment:
default is the default, but the seamless skin doesn't look as nice for unit tests, since it causes the test results at the bottom to move around as content is added. I could be convinced that it's better to use the default skin and deal with the moving around, though.